dependencies/bullseye/msgpack/changelog

```diff
@@ -0,0 +1,5 @@
+msgpack (1.4.2-1) UNRELEASED; urgency=medium
```

Comment:
err, one thing I didn't notice. the package needs to be called `ruby-msgpack` in the changelog

sorry!